chall08/vkurkela.c

```diff
@@ -0,0 +1,40 @@
+int ft_ie_except_after_c(char *str)
+{
+    int i;
+    int len;
+    int exc;
+    
+    i = 0;
+    len = 0;
+    exc = 1;
+    while (str[i++] != '\0')
+        len++;
+    i = 0;
+    while (str[i] != '\0')
+    {
+        if (str[i] == 'e' && (len - (i + 1)) >= 1)
```

Comment:
The condition would be more clear as i + 1 < len or i < len - 1

Reply:
This would be more clear definitely. ^^for the above comment the point why I assingned to 1 at the begin was that if none of 'ie' or 'ei' appear in a word. If either one appears in a word I'm assigning it to 0. I know this isn't the most delicate solution